### DISCUSSION TITLE
一份简明的 Markdown 笔记与教程 — 码志

### DISCUSSION THREAD

**Discussion by mzlogin** [mzlogin/blog-comments#207] [Announcements]
https://mazhuang.org/2018/09/06/markdown-intro/ 

 为部门内知识分享准备的素材，记录了 Markdown 的优点、应用场景和编辑工具，介绍了标准语法与扩展语法，以及一些应用 Markdown 的奇技淫巧。个人使用 Markdown 的经验持续补充中，最新完整版请参见

**Comment by yizibi** [comment]
md还可以做PPT跟流程图,算是长见识了

**Comment by lisongs1995** [comment]
test

**Comment by gttiankai** [comment]
markdown 无所不能

**Comment by zxning** [comment]
👍以后有问题就直接参考了

**Comment by ghost** [comment]
你好，请问下使用了你的模板，但是首页的简介把全文都显示出来了，还有后台有你的仓库，麻烦你了。https://greatplum.github.io//

**Comment by mzlogin** [comment]
> @greatplum
> 你好，请问下使用了你的模板，但是首页的简介把全文都显示出来了，还有后台有你的仓库，麻烦你了。https://greatplum.github.io//

我去看了下，没你说的问题啊。

**Comment by 17607189686** [comment]
1

**Comment by x13-caesar** [comment]
##test
----
>>test

**Comment by ychost** [comment]
我添加了 mathJax.js 和 mermaid.js 实现了数学公式和流程图

**Comment by mzlogin** [comment]
> @ychost
> 我添加了 mathJax.js 和 mermaid.js 实现了数学公式和流程图

哈哈哈 厉害了~这个我也折腾过，这个博客里现在也支持的哈，见 https://mazhuang.org/wiki/markdown/#mermaid

**Comment by dongbisheng** [comment]
个人感觉比较遗憾的一点是各平台可能采用不同语言实现的 Markdown 解析引擎，或采用同一解析引擎的不同版本，而且可能有不同程度的定制与扩展，这导致在不同平台上使用 Markdown 写作时体验并不完全一致。不过幸好对于大家公认的一些标准语法，各家都是支持的。


这明明是优点好吧

**Comment by mzlogin** [comment]
> @dongbisheng
> 个人感觉比较遗憾的一点是各平台可能采用不同语言实现的 Markdown 解析引擎，或采用同一解析引擎的不同版本，而且可能有不同程度的定制与扩展，这导致在不同平台上使用 Markdown 写作时体验并不完全一致。不过幸好对于大家公认的一些标准语法，各家都是支持的。
> 
> 
> 这明明是优点好吧

可能各人的痛点不同吧。

我的看法是如果有一套完善且统一的标准，而且各解析引擎都按这个标准来，那能在不同平台上得到相同的编辑体验，而无须花精力去关注和学习细微差异，体验会更好。

**Comment by MichaelZhou111** [comment]
不错

**Comment by xiaoape** [comment]
很详细

**Comment by damon-dai** [comment]
123

**Comment by xjtu-blacksmith** [comment]
博主好！发现你的示例代码中有几个标题混进目录里了，有点影响效果 :upside_down_face:

**Comment by loylosky** [comment]
nice!

**Comment by ermaozi** [comment]
博主你好, 用你的这个模板怎么用markdown画流程图?

**Comment by mzlogin** [comment]
> 博主你好, 用你的这个模板怎么用markdown画流程图?

这个链接里有演示流程图和时序图等等：https://mazhuang.org/wiki/markdown/

它对应的源码在：https://raw.githubusercontent.com/mzlogin/mzlogin.github.io/master/_wiki/markdown.md

简而言之，就是在头部加上

```
flow: true
```

然后在正文里就可以画流程图了。

**Comment by ermaozi** [comment]
> > 博主你好, 用你的这个模板怎么用markdown画流程图?
> 
> 这个链接里有演示流程图和时序图等等：https://mazhuang.org/wiki/markdown/
> 
> 它对应的源码在：https://raw.githubusercontent.com/mzlogin/mzlogin.github.io/master/_wiki/markdown.md
> 
> 简而言之，就是在头部加上
> 
> ```
> flow: true
> ```
> 
> 然后在正文里就可以画流程图了。

非常感谢!

**Comment by Mints2736** [comment]
# ding

**Comment by mzlogin** [comment]
> @xjtu-blacksmith
> 博主好！发现你的示例代码中有几个标题混进目录里了，有点影响效果 :upside_down_face:

这个问题刚刚解决啦~你再看看 :smile:

**Comment by xjtu-blacksmith** [comment]
> @mzlogin
> > @xjtu-blacksmith
> > 博主好！发现你的示例代码中有几个标题混进目录里了，有点影响效果 :upside_down_face:
> 
> 这个问题刚刚解决啦~你再看看 :smile:

:smile::thumbsup:
